java/overrides.js

```diff
@@ -53,4 +53,6 @@ module.exports = {
       
     // TODO verify and enable
     'should send agentRunId': {skipEmit: true},
+    'appium iOS nav bar check regio': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9214167)
```suggestion
    "appium iOS nav bar check regio": {skipEmit: true},
```